src/master-playlist-controller.js

```diff
@@ -328,6 +331,30 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
+  getPlaylistAttributes_() {
```

Comment:
If this is used in videojs-contrib-hls, should we remove the "private" underscore?